bench/Autofac.Benchmarks/Autofac.Benchmarks.csproj

```diff
@@ -28,13 +28,13 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
```

Comment:
I can't make a comment at the top of this file, but if the `Autofac.BenchmarkProfiling.csproj` changes to net6, this project does, too.

Reply:
Yes Done